projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -71,6 +76,14 @@
 
 public class Client extends AbstractClient implements IClient {
 
+   public class Shape {
```

Comment:
That was test code for the reflections library. Forgot to delete; sorry. Please remove.